Resources/Prototypes/Nyanotrasen/Entities/Structures/Research/sophicscribe.yml

```diff
@@ -28,6 +28,12 @@
     - Common
     - Science
   - type: Psionic
+    removable: false
+  - type: InnatePsionicPowers
+    powersToAdd:
+      - XenoglossyPower
+      - PsychognomyPower
+      - NoosphericZapPower
```

Comment:
Yea I'm not going to question it. This is sufficiently strange that I'm sure you have a particular reason to do this. Carry on here.